CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+## How to contribute to Decidim
+
+We're really glad you're reading this, because we need volunteer developers to help this project come to fruition.
+
+If you haven't already, come find us in [Gitter](https://gitter.im/decidim/decidim). We want you working on things you're excited about.
+
+#### **Did you find a bug?**
+
+* **Do not open up a GitHub issue if the bug is a security vulnerability in Decidim**, and instead send us an email to security [at] decidim.org.
+
+* **Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/decidim/decidim/issues).
+
+* If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/decidim/decidim/issues/new). Be sure to include a **title and clear description**, as much relevant information as possible, and a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+
+#### **Did you write a patch that fixes a bug?**
+
+* Open a new GitHub pull request with the patch.
+
+* Ensure the PR description clearly describes the problem and solution. Include the relevant issue number if applicable.
+
+#### **Did you fix whitespace, format code, or make a purely cosmetic patch?**
+
+Changes that are cosmetic in nature and do not add anything substantial to the stability, functionality, or testability of Decidim will generally not be accepted.
+
```

Comment:
it's from Rails, they also have an issue with the rationale behind this: 
https://github.com/rails/rails/pull/13771#issuecomment-32746700
https://github.com/rails/rails/blob/master/CONTRIBUTING.md

Reply:
I'm aware, I just don't think it applies to us.

Rails has almost 5K all time contributors and 750 open PR's, it's an unusually highly active project for external contributions. I'm sure part of why they introduced that rule is because they were struggling to handle that.

In Decidim we don't have that problem and, at least since I'm around, we've been following something closer to the "no patch is too small" philosophy :)

Reply:
I'd like some more feedback regarding the cosmetic changes. @decidim/developers what do you think about these lines?

Reply:
I'm 👍 to what @deivid-rodriguez says here, at least for now. Apart from being properly formatted, our code can have some refactors and this part can seem like we won't accept refactors

Reply:
Ok, I'll remove this lines, pefect!